platform/mellanox/mlnx-platform-api/sonic_platform/utils.py

```diff
@@ -285,3 +288,22 @@ def wait_until(predict, timeout, interval=1, *args, **kwargs):
         time.sleep(interval)
         timeout -= interval
     return False
+
+
+class DbUtils:
+    db_instances = {}
+
+    @classmethod
+    def get_db_instance(cls, db_name, **kargs):
```

Comment:
@Junchao-Mellanox just to support mock test we are adding this?

Reply:
We add this to access STATE_DB to see if the cable is under software control or firmware control.